### DISCUSSION TITLE
Bad escape  error - Manager not starting

### DISCUSSION THREAD

**Discussion by mlattari** [swar/Swar-Chia-Plot-Manager#1287] [Q&A] [answered]
Trying to run the manager i get: 

```
File "/chia/swar-plot-manager/Swar-Chia-Plot-Manager/swar-venv/lib/python3.9/site-packages/regex/_regex_core.py", line 1736, in _compile_replacement
    raise error("bad escape \\%s" % ch, source.string, source.pos)
```

How to solve it?

**Comment by schinzelh** [comment, marked as answer]
Looks like there is a issue with the dateparser version used and the recent regexp library.
Update requirements.txt dateparser to 1.1.1, run `pip install -r requirements.txt` and try starting the manager again.

See https://github.com/scrapinghub/dateparser/issues/1052

**Reply by mlattari** [reply]
Thank You! It worked :-)
